portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro

```diff
@@ -167,7 +171,7 @@ definition {
 			AssertElementPresent(locator1 = "UserBar#USER_AVATAR_IMAGE");
 		}
 		else {
-			User.loginPG(
+			User.firstLoginUI(
 				password = "${password}",
 				samlEnabled = "true",
```

Comment:
Remove useless parameters when using User.firstLoginUI, for example **samlEnabled = "true",** doesn't exist under macro User.firstLoginUI.

Reply:
Thanks I remove this unused property!